scripts/run_validator.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
 
-tag=latest
+tag=add-support-for-first-item-in-list-source
```

Comment:
Did Andrews suggest not work?

Reply:
This is Andrew's suggestion. It required a small change to code.